LayoutTests/editing/selection/iframe-select.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<iframe id="iframe" style="width: 100px; height: 100px; position: absolute; top: 50px; left: 50px" frameBorder=0 srcdoc="<div></div>"></iframe>
```

Comment:
This test needs to add `will-change: transform` to the style to exercise the correct code path.

Reply:
Done.